book/intro.md

```diff
@@ -1,11 +1,158 @@
-# Welcome to your Jupyter Book
+# Mastering Geospatial Visualizations with GMT/PyGMT
 
-This is a small sample book to give you a feel for how book content is
-structured.
-It shows off a few of the major file types, as well as some sample content.
-It does not go in-depth into any particular topic - check out [the Jupyter Book documentation](https://jupyterbook.org) for more information.
+## Welcome to the AGU24 GMT/PyGMT workshop 🥳
 
-Check out the content pages bundled with this sample book to see more.
+This Jupyter book 📖 contains [GMT](https://docs.generic-mapping-tools.org/6.5) and
+[PyGMT](https://www.pygmt.org/v0.13.0) tutorials for producing maps 🗺️ and doing
+geospatial data processing 🌐
 
-```{tableofcontents}
+::::{grid} 3
+
+:::{grid-item-card} Tutorial 1 - First figure + Subplots / layout
+:link: ./first-figure.html
+by [Jing-Hui Tong](https://orcid.org/0009-0002-7195-3071)
++++
+{bdg-primary}`pygmt`
+{bdg-primary-line}`coast`
+{bdg-primary-line}`grdimage`
+{bdg-primary-line}`makecpt`
+{bdg-primary-line}`colorbar`
+{bdg-primary-line}`subplot`
+{bdg-secondary-line}`earth_relief`
+{bdg-secondary-line}`earth_age`
+:::
+
+:::{grid-item-card} Tutorial 2 - Integration with the Scientific Python Ecosystem: Pandas / GeoPandas
+:link: ./tut2_spec_pd_gpd.ipynb.html
+by [Yvonne Fröhlich](https://orcid.org/0000-0002-8566-0619)
++++
+{bdg-primary}`pygmt`
+{bdg-primary-line}`histogram`
+{bdg-primary-line}`legend`
+{bdg-primary-line}`plot`
+{bdg-secondary-line}`japan_quakes`
+{bdg-info}`pandas`
+{bdg-info}`geopandas`
+{bdg-info-line}`airbnb`
```

Comment:
Not sure about this; I would prefer to include "choropleth map"
```suggestion
{bdg-info-line}`choropleth map`
```

Reply:
Yeah, I could use some advice on what tags/badges to add. My current thought process is to include:

- bdg-primary: pygmt or gmt
- bdg-primary-line: GMT module used (e.g. `coast`, `plot`, etc)
- bdg-info: External library used (e.g. `pandas`, `xarray`)
- bdg-secondary-line: GMT remote-dataset used (e.g. `earth_relief`, `earth_age`, etc)

The ``` {bdg-info-line}`airbnb` ``` here was intended to show external non GMT remote-datasets used, but I do agree that having AirBnB here looks weird. We could maybe add some tags for concepts being taught such as `choropleth-map` as you suggested, or `scatterplot`, etc.

Ref: color of badges/tags: https://sphinx-design.readthedocs.io/en/sbt-theme/badges_buttons.html#badges

Reply:
Yep, I got your main idea behind the structure of these tags / badges and I definitely like this idea!  And I think mentioning special concept / specific visualization is a good addition.  

Even the file contains "airbnb" currently the column “population” is used for the choropleth-map (there are a lot of different columns / quantities in this dataset).